cfe_internal/enterprise/templates/runalerts.php.mustache

```diff
@@ -9,11 +9,15 @@ if(PHP_SAPI != 'cli')
 
 while(1 == 1)
 {
-  print("Checking for sql alerts");
+  status=0;
```

Comment:
all variables should start with $, $status

Reply:
Thanks.

Reply:
Updated, squashed.